src/providers/docker-images/codebuild/linux-arm64/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE="public.ecr.aws/lts/ubuntu:20.04"
+ARG BASE_IMAGE="public.ecr.aws/lts/ubuntu:latest"
```

Comment:
Is upgrading required for this change? We should ideally keep this to a separate PR as it may be a breaking change for some people.

Reply:
not required, but as more recent LTS is already there, I thought it's good always to have it the latest available LTS
If there's some breaking change, can't they put in a local older version of it?

Reply:
Introducing breaking changes should be documented. Yes, they can input an older version, but I don't want that requirement to be a surprise. That's why it should be a separate PR so it can be discussed and documented.

Reply:
no problem, will sepearate